### PR TITLE
chore(deps): upgrade mocha in eslint-plugin-fxa

### PR DESCRIPTION
**Pull request by chenba**
This commit also resolves `flat` in pm2's dependency graph to a newer version.

### CODE REVIEW

**Comment by julianpoy** [comment]
Any chance this could wait until after #15261?

**Comment by chenba** [comment]
> Any chance this could wait until after #15261?

@julianpoy are you worried about merge conflicts or something else?

File: packages/eslint-plugin-fxa/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "ramda": "^0.26.1"
   },
   "devDependencies": {
-    "mocha": "6.2.2"
+    "mocha": "10.2.0"
```
Comment (dschom) [inline packages/eslint-plugin-fxa/package.json:15]: Just double checking that we want 10.2.0 and not ^10.2.0. Most of our package.json ref mocha as ^10.0.0.

**Review by dschom — APPROVED** [review]
Looks good to me. Please coordinate with @julianpoy on the merge. A quick glance at the nx PR, and it seems like this PR can land without too much disruption, but perhaps not everything is pushed?

**Comment by julianpoy** [comment]
Apologies, missed this. Feel free to merge - I was delayed a sprint.

**Pull request merged**